telegram/ext/picklepersistence.py

```diff
@@ -105,7 +105,7 @@ def load_singlefile(self) -> None:
                 # For backwards compatibility with files not containing bot data
                 self.bot_data = data.get('bot_data', {})
                 self.conversations = data['conversations']
-        except IOError:
+        except OSError:
```

Comment:
we sure this change is alright?

Reply:
yeah, looked it up, some exceptions were merged into OSError. just below [here](https://docs.python.org/3/library/exceptions.html#OSError.filename2)

Reply:
cool